vignettes/carbon_tracking.Rmd

```diff
@@ -207,21 +204,22 @@ knorr_values <- c(0.45, 0.453, 0.518, 0.468, 0.468, 0.514, 0.449)
 knorr_avg <- mean(knorr_values)
 knorr_minmax <- max(knorr_values) - min(knorr_values)
 
-names <- c("Knorr avg", "1950", "1960", "1970", "1980", "1990", "2000", "2010", "2020")
-values <- c(knorr_avg, AF_1950, AF_1960, AF_1970, AF_1980, AF_1990, AF_2000, AF_2010, AF_2020)
-error <- c(knorr_minmax, 0, 0, 0, 0, 0, 0, 0, 0)
+
+names <- c("Knorr avg", years)
```

Comment:
Minor point: since `names()` is an R function, I would stay away from using this for a variable. It works, but potentially confusing.